src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.html

```diff
@@ -20,6 +20,9 @@
         (activate)="setActiveOutcome(o.id)"
       >
       </clark-outcome>
+      <div class="addButton" >
```

Comment:
What is the reasoning behind the wrapper for the button?

Reply:
we spoke about not have the button just sit there. we it to have a white background to complete the card. so decided to style the div.